src/core/asyncUtils.ts

```diff
@@ -10,8 +10,13 @@ import {
 import { resolveAfter, callAfter } from '../helpers/promises'
 import { TimeoutError } from '../helpers/error'
 
-const DEFAULT_INTERVAL = 50
-const DEFAULT_TIMEOUT = 1000
+let defaultInterval: number | false = 50
+let defaultTimeout: number | false = 1000
+
+function setDefaultWaitOptions({ interval, timeout }: WaitOptions) {
+  defaultInterval = interval ?? defaultInterval
+  defaultTimeout = timeout ?? defaultTimeout
+}
```

Comment:
Do we need some way of resetting the defaults back to the initial values?

I'm just thinking about someone that wants to update the defaults for a single test file so they set it in the `beforeAll` and want to reset it in the `afterAll`.  Because we don't export the initial default values, they would have to just hard code the documented default values and hope we never change them.

The options I see are:
1. export some constants for them to use with `setDefaultWaitOptions`
2. export a `resetDefaultWaitOptions` utility

Or maybe both?  Or maybe there's another option I'm not seeing?

Reply:
I think exporting the constants makes more sense than a function imo

Reply:
I've been wondering about options into `renderHook`? A render helper can then be extracted like normal for shared setup.

Reply:
@mpeyper, can you expand on your idea? I will export initial constants otherwise